builder/Makefile.tracee-container

```diff
@@ -93,7 +93,12 @@ build-alpine-tracee-core: \
 		-t $(ALPINE_TRACEE_CORE_CONTNAME):latest \
 		--build-arg=BTFHUB=0 \
 		--build-arg=FLAVOR=tracee-core \
+		--target tracee-core \
 		.
+		$(CMD_DOCKER) images \
+		--filter "label=AS=tracee-make" \
+		--format "{{.ID}}" \
+		| xargs docker rmi
```

Comment:
nit: xargs $(CMD_DOCKER) for uniformity (and in all other "xargs docker rmi" places).